tuned/admin/admin.py

```diff
@@ -316,6 +317,8 @@ def _action_recommend_profile(self):
 		return True
 
 	def _action_dbus_verify_profile(self, ignore_missing):
+		if not self._async and self._log_level is not None:
+			self._log_capture_start()
```

Comment:
What about initially limiting the timeout to few minutes, let's say 10 and limiting buffers to one log buffer per UID for non root users? Or we can tighten the verify policy.

Reply:
I was thinking the limit could be 2 buffers of size 1MB per non-root users, but your suggestion could work too. I'd prefer adding the limit, rather than tightening the verify policy. Either way, we agreed with Tomas that I'll look into implementing it, so this pull request is waiting on me.

Reply:
No problem, I am OK with your suggestion and proposed buffer limits. My suggestion was just something to start from.